console/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "cypress:ci-build-e2e": "NODE_ENV=test dotenv-flow -- cypress run --browser chrome --config retries=2,video=true,baseUrl=http://localhost:80 --spec 'cypress/e2e/unauthenticated/**/*' --record --parallel --group e2e",
-    "cypress:ci-build-auth": "NODE_ENV=test dotenv-flow -- cypress run --browser chrome --config retries=2,video=true,baseUrl=http://localhost:80 --spec 'cypress/e2e/authenticated/**/*' --record --parallel --group e2e-auth",
+    "cypress:ci-build-auth": "NODE_ENV=test dotenv-flow -- cypress run --browser chrome --config retries=2,video=true,baseUrl=http://127.0.0.1:80 --spec 'cypress/e2e/authenticated/**/*' --record --parallel --group e2e-auth",
```

Comment:
I needed to change the baseUrl in order for console to work when deployed with enterprise using bazel.